### ISSUE TITLE
Accessibility - Move “all of the above” to the end.

### ISSUE THREAD

**Issue by PeaceGeeksProjects** [PeaceGeeksSociety/ServicesAdvisor-2.0#74]
http://admin.jordan.servicesadvisor.org/service-accessibility/all-above
http://admin.jordan.servicesadvisor.org/node/281/edit

The client added a term for "all of the above" into the accessibility taxonomy, then workshop participants used that term.  This doesn't work as expected, cause the other terms are not checked, and therefor wont appear in searches.

We need to remove the All of the Above option, and set all the other values to checked before deleting the term "All of the Above"

**Comment by mackh** [comment]
‏الاتصال بالخط الساخن والزيارة المنزلية المتابعة للاتصال ‏ is the Arabic translation for the english term "All of the Above" this means the same thing as many of the other terms in the Accessibility Vocabulary, so the client has implmented this specifically, we need to validate their intent before we make changes

**Comment by mackh** [comment]
we have removed the "all of the above" option since it doesnt work in search.  We migrated any records with this selection to show all options as checked.

**Comment by PeaceGeeksProjects** [comment]
Closing then...
